### PR TITLE
build(webkit): inlcude libvpx.so.5 into the .zip

### LINKED ISSUE #2447: [BUG] WebKit browser crashes on new page/context on Ubuntu
**Context:**
- Playwright Version: 1.0.2
- Operating System: Ubuntu 20.04 LTS
- Node version: 12.17.0
- Browser: WebKit
- Extra: 
  - I installed the same WebKit dependencies as the [Ubuntu bionic docker file](https://github.com/microsoft/playwright/blob/master/docs/docker/Dockerfile.bionic) except for `libevent-2.1-6` that was replaced by `libevent-2.1-7` and `libvpx5` that was replaced by `libvpx6` on Ubuntu apt servers.
  - I have [Epiphany](https://wiki.gnome.org/Apps/Web), a WebKit-based browser, working fine on my machine.

**Code Snippet**

```javascript
// debug.js
const { webkit } = require('playwright');

(async () => {
    const browser = await webkit.launch();
    const page = await browser.newPage();
    await page.goto('http://whatsmyuseragent.org/');
    await page.screenshot({ path: `example.png` });
    await browser.close();
})();
```

**Describe the bug**
```
 % node ./debug.js
(node:19443) UnhandledPromiseRejectionWarning: Error: Protocol error (Playwright.createContext): Browser has been closed.
    at /home/laurent/projects/myproject/node_modules/playwright-core/lib/webkit/wkConnection.js:102:63
    at new Promise (<anonymous>)
    at WKSession.send (/home/laurent/projects/myproject/node_modules/playwright-core/lib/webkit/wkConnection.js:101:16)
    at WKBrowser.newContext (/home/laurent/projects/myproject/node_modules/playwright-core/lib/webkit/wkBrowser.js:65:65)
    at WKBrowser.<anonymous> (/home/laurent/projects/myproject/node_modules/playwright-core/lib/helper.js:64:31)
    at WKBrowser.newPage (/home/laurent/projects/myproject/node_modules/playwright-core/lib/browser.js:30:36)
    at /home/laurent/projects/myproject/debug/debug.js:5:32
(node:19443) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:19443) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

The code snippet is working fine with `chromium` and `firefox` browsers.

### PR DESCRIPTION
**Pull request by yury-s**
Ubuntu 20.04 comes with libvpx6 so we need to provide libvpx5 ourselves.

Drive-by: make archive.sh work when checkout is a symlink.

Fixes: #2447

### CODE REVIEW

File: browser_patches/webkit/archive.sh [hunk 1]
```diff
@@ -62,6 +62,8 @@ createZipForLinux() {
     LD_LIBRARY_PATH="$PWD/WebKitBuild/WPE/DependenciesWPE/Root/lib" ldd WebKitBuild/WPE/Release/bin/MiniBrowser | grep -o '[^ ]*WebKitBuild/WPE/[^ ]*' | xargs cp -t $tmpdir
     LD_LIBRARY_PATH="$PWD/WebKitBuild/WPE/DependenciesWPE/Root/lib" ldd WebKitBuild/WPE/Release/bin/WPENetworkProcess | grep -o '[^ ]*WebKitBuild/WPE/[^ ]*' | xargs cp -t $tmpdir
     LD_LIBRARY_PATH="$PWD/WebKitBuild/WPE/DependenciesWPE/Root/lib" ldd WebKitBuild/WPE/Release/bin/WPEWebProcess | grep -o '[^ ]*WebKitBuild/WPE/[^ ]*' | xargs cp -t $tmpdir
+    # Copy libvpx.so.5 as Ubuntu 20.04 comes with libvpx.so.6
+    ldd /home/yurys/WebKit/WebKitBuild/GTK/Release/bin/MiniBrowser | grep -o '[^ ]*\/libvpx.so.5[^ ]*' | xargs cp -t $tmpdir
```
Comment (aslushnikov) [inline browser_patches/webkit/archive.sh:66]: This absolute path won't work on bots

File: browser_patches/webkit/archive.sh [hunk 2]
```diff
@@ -73,6 +75,8 @@ createZipForLinux() {
     cp -t $tmpdir ./WebKitBuild/GTK/Release/bin/MiniBrowser ./WebKitBuild/GTK/Release/bin/WebKit*Process
     # copy all relevant shared objects
     LD_LIBRARY_PATH="$PWD/WebKitBuild/GTK/DependenciesGTK/Root/lib" ldd WebKitBuild/GTK/Release/bin/MiniBrowser | grep -o '[^ ]*WebKitBuild/GTK/[^ ]*' | xargs cp -t $tmpdir
+    # Copy libvpx.so.5 as Ubuntu 20.04 comes with libvpx.so.6
+    ldd /home/yurys/WebKit/WebKitBuild/GTK/Release/bin/MiniBrowser | grep -o '[^ ]*\/libvpx.so.5[^ ]*' | xargs cp -t $tmpdir
```
Comment (aslushnikov) [inline browser_patches/webkit/archive.sh:79]: absolute path here as well

File: browser_patches/webkit/archive.sh (hunk 1 above)
Comment (yury-s) [reply browser_patches/webkit/archive.sh]: Fixed.

File: browser_patches/webkit/archive.sh (hunk 2 above)
Comment (yury-s) [reply browser_patches/webkit/archive.sh]: Fixed

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**